nservicebus/pipeline/manipulate-with-behaviors_registernew_core_[6.0,).partial.md

```diff
@@ -7,4 +7,4 @@ Behaviors can also be registered from a `Feature` as shown below:
 
 snippet: RegisterBehaviorFromFeature
 
-WARNING: Behaviors are only created once and the same instance is reused on every invocation of the pipeline. Consequently, every behavior dependency will also behave as a singleton, even if a different option was specified when registering it in the DI container.
+WARNING: Behaviors are only created once and the same instance is reused on every invocation of the pipeline. Consequently, every behavior dependency will also behave as a singleton, even if a different option was specified when registering it in the DI container. Furthermore, the behavior as well as the dependencies called during the invocation phase needs to be concurrency safe.
```

Comment:
If possible, then would be great to explain how to verify if it's safe. Do we have any guidelines in other place?